qhub/template/stages/02-infrastructure/azure/main.tf

```diff
@@ -26,4 +38,8 @@ module "kubernetes" {
       max_size      = config.max_nodes
     }
   ]
+
+  depends_on = [
+    azurerm_resource_group.resource_group,
```

Comment:
Not needed since `resource_group_name      = azurerm_resource_group.resource_group.name` is already specified here.

Reply:
oh definitely, Thanks Chris